packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -132,7 +132,10 @@ depth = model.infer_image(raw_img) # HxW raw depth map in numpy
 const diffusers_default = (model: ModelData) => [
 	`from diffusers import DiffusionPipeline
 
-pipeline = DiffusionPipeline.from_pretrained("${model.id}")`,
+pipeline = DiffusionPipeline.from_pretrained("${model.id}")
```

Comment:
```suggestion
pipe = DiffusionPipeline.from_pretrained("${model.id}")
```
like in transformers? (unsure, up to you, no big difference lol)

Reply:
Not a big deal I am okay with `pipe`, too.

Reply:
`pipe` looks good! I'll update it in all the other snippets too.